x-pack/test/rule_registry/security_and_spaces/tests/basic/update_alert.ts

```diff
@@ -0,0 +1,266 @@
+  const SECURITY_SOLUTION_ALERT_ID = '020202';
+  const SECURITY_SOLUTION_ALERT_INDEX = '.alerts-security-solution';
+  const ALERT_VERSION = Buffer.from(JSON.stringify([0, 1]), 'utf8').toString('base64'); // required for optimistic concurrency control
+
+  const getAPMIndexName = async (user: User) => {
+    const {
+      body: indexNames,
+    }: { body: { index_name: string[] | undefined } } = await supertestWithoutAuth
+      .get(`${getSpaceUrlPrefix(SPACE1)}${ALERTS_INDEX_URL}`)
+      .auth(user.username, user.password)
+      .set('kbn-xsrf', 'true')
+      .expect(200);
+    const observabilityIndex = indexNames?.index_name?.find(
+      (indexName) => indexName === APM_ALERT_INDEX
+    );
+    expect(observabilityIndex).to.eql(APM_ALERT_INDEX); // assert this here so we can use constants in the dynamically-defined test cases below
+  };
+
+  const getSecuritySolutionIndexName = async (user: User) => {
+    const {
+      body: indexNames,
+    }: { body: { index_name: string[] | undefined } } = await supertestWithoutAuth
+      .get(`${getSpaceUrlPrefix(SPACE1)}${ALERTS_INDEX_URL}`)
+      .auth(user.username, user.password)
+      .set('kbn-xsrf', 'true')
+      .expect(200);
+    const securitySolution = indexNames?.index_name?.find(
+      (indexName) => indexName === SECURITY_SOLUTION_ALERT_INDEX
+    );
+    expect(securitySolution).to.eql(SECURITY_SOLUTION_ALERT_INDEX); // assert this here so we can use constants in the dynamically-defined test cases below
+  };
+
+  describe('Alert - Update - RBAC - spaces', () => {
+    before(async () => {
+      await getSecuritySolutionIndexName(superUser);
+      await getAPMIndexName(superUser);
+    });
+
+    before(async () => {
+      await esArchiver.load('x-pack/test/functional/es_archives/rule_registry/alerts');
+    });
+
+    after(async () => {
+      await esArchiver.unload('x-pack/test/functional/es_archives/rule_registry/alerts');
+    });
+
+    it('should return a 404 when superuser accesses not-existent alert', async () => {
+      await supertestWithoutAuth
+        .post(`${getSpaceUrlPrefix()}${TEST_URL}`)
+        .auth(superUser.username, superUser.password)
+        .set('kbn-xsrf', 'true')
+        .send({
+          ids: ['this id does not exist'],
+          status: 'closed',
+          index: APM_ALERT_INDEX,
+          _version: Buffer.from(JSON.stringify([0, 1]), 'utf8').toString('base64'),
+        })
+        .expect(404);
+    });
+
+    it('should return a 404 when superuser accesses not-existent alerts as data index', async () => {
+      await supertestWithoutAuth
+        .post(`${getSpaceUrlPrefix()}${TEST_URL}`)
+        .auth(superUser.username, superUser.password)
+        .set('kbn-xsrf', 'true')
+        .send({
+          ids: [APM_ALERT_ID],
+          status: 'closed',
+          index: 'this index does not exist',
+          _version: Buffer.from(JSON.stringify([0, 1]), 'utf8').toString('base64'),
+        })
+        .expect(404);
+    });
```

Comment:
Nit: you can remove the superuser-specific tests here and add them to the group for all users instead (see comment below)

```suggestion
```